lib/ProductOpener/Config_off.pm

```diff
@@ -406,6 +406,9 @@ fr:levure
 fr:levures
 en:honeys
 en:vinegars
+en:pet food
+en:non food products
```

Comment:
As Stephane said, please use hyphens instead of spaces.

Reply:
done